mypy/errorcodes.py

```diff
@@ -3,19 +3,25 @@
 These can be used for filtering specific errors.
 """
 
-from typing import List
+from typing import Dict, List, Set
 from typing_extensions import Final
 
 
 # All created error codes are implicitly stored in this list.
 all_error_codes = []  # type: List[ErrorCode]
```

Comment:
This list seemed so far unused, and for my use case was not the ideal data structure, so I added the below dict. If it's preferred I can change to adding the error codes here and iterating through.